flexmeasures/ui/views/assets/forms.py

```diff
@@ -42,6 +42,7 @@ class AssetForm(FlaskForm):
         "Sensors to show as KPIs (JSON)",
         default="[]",
     )
+    external_id = StringField("ID for this asset in another system", default="")
```

Comment:
I guess I would just name this `"External ID"` so that the user can straightforwardly map the field description to what they later see in the UI/API/CLI. Does the `StringField` allow setting a description, too? If so, maybe this sentence (with a period) could serve as a tooltip description instead.

Alternatively, `"External ID used by other systems"` to get the best of both worlds.

Also, we're not yet marking clearly which fields are optional. Perhaps a separate issue.